featuretools/primitives/standard/rolling_transform_primitive.py

```diff
@@ -618,34 +589,17 @@ def __init__(self, window_length=3, gap=0, min_periods=0):
 
     def get_function(self):
         def rolling_count(datetime):
+            def _pandas_count(series):
+                return series.count()
```

Comment:
I followed what was already in RollingStd but I feeling like we could simplify this by just passing in the lamda function in `apply_rolling_agg_to_series` instead.